tests/plugins/test_kubejobs.py

```diff
@@ -29,11 +30,75 @@ def test_visualize_grafana():
             Returns:
                 None
     """
-    job_id = create_job(MANAGER_URL)
+    job_id = create_job(MANAGER_URL,1)
     grafana_url = wait_for_grafana_url_generation(VISUALIZER_URL, job_id)
     assert requests.get(grafana_url).ok
     stop_job(manager_url=MANAGER_URL, job_id=job_id)
 
+def test_scalling_up():
+    """ Tests if the Controlling is able to scale
+        Arguments:
+            None
+        Returns:
+            None
+    """
+    INITIAL_REPLICAS = 1
+    job_id = create_job(MANAGER_URL,2)
+    wait_for_job_complete(MANAGER_URL,job_id)
+    response = requests.get(MANAGER_URL + '/submissions/{}'.format(job_id))
+    replicas = response.json().get('final_replicas')
+    print(response.json())
+    assert int(replicas) > INITIAL_REPLICAS
+    stop_job(manager_url=MANAGER_URL, job_id=job_id)
+
+def test_scalling_down():
+    """ Tests if the Controlling is able to scale
+        Arguments:
+            None
+        Returns:
+            None
+    """
+    INITIAL_REPLICAS = 10
+    job_id = create_job(MANAGER_URL,3)
+    wait_for_job_complete(MANAGER_URL,job_id)
+    response = requests.get(MANAGER_URL + '/submissions/{}'.format(job_id))
+    replicas = response.json().get('final_replicas')
+    print(response.json())
+    assert int(replicas) < INITIAL_REPLICAS
+    stop_job(manager_url=MANAGER_URL, job_id=job_id)
+
+def test_monitor_metrics():
```

Comment:
Right from the beginning you say two checks are made. Considering this, I would split this test into two test cases. One checking if the monitor is providing metrics, and the other if the simple report matches the detailed one. If it is not what happens in the actual test, rename it according to what it actually makes, which for me seams to be only the second part.